File: 백준/Bronze/9063. 대지/대지.cc
#include <iostream>
using namespace std;

int main() {
    ios::sync_with_stdio(false); cin.tie(NULL); cout.tie(NULL);
    
    int n;
    int min_x = 10001, max_x = -10001;
    int min_y = 10001, max_y = -10001;   
    cin >> n;

    for (int i = 0; i < n; i++) {
        int x, y;
        cin >> x >> y;

        if (x > max_x)
            max_x = x;
        if (x < min_x)
            min_x = x;

        if (y > max_y)
            max_y = y;
        if (y < min_y)
            min_y = y;
    }

    cout << (max_x - min_x) * (max_y - min_y) << "\n";
    
    return 0;
}

File: 백준/Bronze/9063. 대지/대지.py
import sys

n = int(sys.stdin.readline())
max_x, min_x = -10001, 10001
max_y, min_y = -10001, 10001

for i in range(n):
    x, y = map(int, sys.stdin.readline().split())

    if x > max_x:
        max_x = x
    if x < min_x:
        min_x = x

    if y > max_y:
        max_y = y
    if y < min_y:
        min_y = y

sys.stdout.write(f"{(max_x - min_x) * (max_y - min_y)}\n")